package.json

```diff
@@ -2,14 +2,15 @@
   "name": "plumber",
   "license": "AGPL-3.0",
   "scripts": {
+    "postinstall": "npm run graphql-codegen",
     "all": "concurrently -c auto",
     "setup": "npm run -w backend setup",
     "dev:frontend": "npm run -w frontend dev",
     "dev:backend": "npm run -w backend dev",
     "dev:worker": "npm run -w backend worker",
     "dev": "npm run all --kill-others npm:dev:*",
-    "build:frontend": "npm run -w frontend build",
-    "build:backend": "npm run -w backend build",
+    "graphql-codegen": "graphql-codegen --config gql-codegen.ts",
+    "gqlc": "npm run graphql-codegen",
```

Comment:
nitpick: seems like there's a larger number of commands to run now before getting the dev environment ready, might it make sense to introduce a single `npm run predev` command for convenience that can handle codegen, DB migration etc?

Reply:
Hmmm.... I think anything that tightens the development loop (time between code change -> dev server change) would be good.

If you agree with my thinking, then adding more actions into `dev` command would not be ideal, since it increases the time between the `npm run dev` command and server startup. Although not significant in a single instance, the delay adds up if we have to restart our servers while writing code (e.g. debugging crashes etc).

However I do see the problem you're pointing out. A alternative approach might be to set up watch on `schema.graphql` and rerun  codegen automatically, and watch on db migrations directory and run migrations automatically. (@pregnantboy doesn't agree with this though - I think he up watch on graphql for openspace before and it was flakey?)